src/components/store/OrderSummary/index.tsx

```diff
@@ -73,45 +69,56 @@ const OrderSummary = ({
         onClose={() => setCancelModalOpen(false)}
         cancelOrder={cancelOrder}
       />
-      {items.map(item => (
-        <Link
-          href={`${config.store.itemRoute}${item.option.item.uuid}`}
-          key={item.uuid}
-          className={styles.itemInfo}
-        >
-          <div className={styles.image}>
-            <Image
-              src={getDefaultOrderItemPhoto(item)}
-              style={{ objectFit: 'cover' }}
-              sizes="9.375rem"
-              alt="Store item picture"
-              fill
-            />
-          </div>
-          <div className={styles.itemSummary}>
-            <Typography variant="h4/bold">{item.option.item.itemName}</Typography>
-            <div className={styles.label}>
-              <Typography variant="h5/bold">Price: </Typography>
-              <Diamonds
-                count={item.option.price * item.quantity}
-                discount={item.salePriceAtPurchase}
+      {items.map(item => {
+        if (item.fulfilled && orderStatus === OrderStatus.PLACED) {
```

Comment:
It might help to add a comment explaining why or in what scenario this is necessary